### ISSUE TITLE
Make messages shorter and more formal

### ISSUE THREAD

**Issue by ffoodd** [ffoodd/a11y.css#141]
In order to make using a11y.css comfortable, motivating and inspiring, messages should be enjoyable and pleasant.

Thanks to Johan Ramon for the comment.

**Comment by ffoodd** [comment]
Probably need an editorial guideline for this one, describing tone and goal for example.

**Comment by HugoGiraudel** [comment]
I think most messages should be reviewed. I like the pleasant tone of a11y.css messages but I feel like it makes it appear less serious than it actually is. I would love to see something a bit more formal.
